lakesoul-flink/pom.xml

```diff
@@ -154,6 +154,11 @@
             <version>${flink.version}</version>
             <scope>${local.scope}</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.flink</groupId>
+            <artifactId>flink-csv</artifactId>
+            <version>${flink.version}</version>
+        </dependency>
```

Comment:
Is flink-csv neccasary?

Reply:
flink sqlclient exports parquet as csv file by flink-csv for k8s test

Reply:
> Is flink-csv neccesary?

flink sqlclient exports parquet as csv file by flink-csv for k8s test